### PR TITLE
Improvements 

### PR DESCRIPTION
**Pull request by emidev98**
-> FEAT:  use new [plugin-not-found](https://oclif.io/docs/plugins#useful-plugins)
-> FEAT: [**use** and **prepack:use**](https://github.com/terra-money/terrain/pull/32/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R29-R30) scripts to package.json
-> FIX: path to [help plugin](https://github.com/terra-money/terrain/pull/32/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R21)
-> CHORE: re-order [package.json](https://github.com/terra-money/terrain/blob/8de647fecf7b3ef31e9b919a4bae5db326571b54/package.json)
-> DOCS: how to [run the project locally](https://github.com/terra-money/terrain/pull/32/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R705)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ Terrain will help you:
 * [Usage](#usage)
 * [Commands](#commands)
 * [Use main branch in local.](#use-main-branch-in-local)
+* [How to run project locally](#how-to-run-project-locally)
```
Comment (ericHgorski) [inline README.md:36]:
```suggestion
* [Running Terrain locally](#running-terrain-locally)
```

File: README.md [hunk 2]
```diff
@@ -700,3 +701,26 @@ Inside the project folder execute [install](https://docs.npmjs.com/cli/v6/comman
 ```
 
 > Take in consideration that the process documented before sometimes will contain fixes and new features but is still being tested.
+
+# How to run the project locally
```
Comment (ericHgorski) [inline README.md:705]:
```suggestion
# Running Terrain locally
```

**Comment by emidev98** [comment]
Applied 🙃

**Review by ericHgorski — APPROVED** [review]

**Comment by ericHgorski** [comment]
Great stuff @emidev98 ! Just added the `oclif.manifest` (produced by prepack) to `.gitignore`. Will be great to get this out so the help command works once more, we should do a release soon @octalmage.

**Pull request merged**